### PR TITLE
Bug 2012796: Add LivenessProbe to Image Service

### PR DESCRIPTION
**Pull request by rollandf**
# Assisted Pull Request

## Description

Add LivenessProbe to Image Service

## List all the issues related to this PR

- [ ] New Feature
- [x] Bug fix
- [ ] Tests
- [ ] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [ ] Automation (CI, tools, etc)
- [ ] Cloud
- [x] Operator Managed Deployments
- [ ] None

## How was this code tested?

- [ ] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [ ] Waiting for CI to do a full test run
- [ ] Manual (Elaborate on how it was tested)
- [x] No tests needed

## Assignees

/cc @carbonin 
/cc @

## Checklist

- [x] Title and description added to both, commit and PR.
- [x] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [x] Reviewers have been listed
- [x] This change does not require a documentation update (docstring, `docs`, README, etc)
- [x] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- [ ] Are the title and description (in both PR and commit) meaningful and clear?
- [ ] Is there a bug required (and linked) for this change?
- [ ] Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

**Comment by rollandf** [comment]
/hold cancel
image-service side merged:
https://github.com/openshift/assisted-image-service/pull/36

File: internal/controller/controllers/agentserviceconfig_controller.go [hunk 1]
```diff
@@ -765,6 +765,16 @@ func (r *AgentServiceConfigReconciler) newImageServiceDeployment(ctx context.Con
 				},
 			},
 		},
+		LivenessProbe: &corev1.Probe{
+			InitialDelaySeconds: 30,
```
Comment (carbonin) [inline internal/controller/controllers/agentserviceconfig_controller.go:769]: Why the delay?

Comment (rollandf) [reply internal/controller/controllers/agentserviceconfig_controller.go:769]: I took the same as assisted-service:
https://github.com/openshift/assisted-service/blob/master/internal/controller/controllers/agentserviceconfig_controller.go#L915

**Pull request merged**